inst/include/stan_meta_header.hpp

```diff
@@ -0,0 +1 @@
+// Insert all #include<foo.hpp> statements here
```

Comment:
Do we need this?

Reply:
I'm not sure, once we get things to work on Jaume's computer we can take things out and see if it stops working.